src/os/bluestore/bluestore_tool.cc

```diff
@@ -219,6 +227,62 @@ void inferring_bluefs_devices(vector<string>& devs, std::string& path)
   }
 }
 
+void bluefs_import(string input_file,
+                   CephContext *cct,
+                   const string& path,
+                   const vector<string>& devs)
+{
+    int r;
+    std::ifstream f;
+    f.open(input_file.c_str(), std::ifstream::binary);
+    if (!f.is_open()) {
```

Comment:
nit, 
```suggestion
    if (f) {
```

would suffice.